app/src/main/java/com/codesoom/assignment/services/TaskIdGenerator.java

```diff
@@ -0,0 +1,21 @@
+package com.codesoom.assignment.services;
+
+import org.springframework.stereotype.Service;
+
+/**
+ * 할 일 번호 생성을 담당합니다.
+ */
```

Comment:
번호라는 표현보다 더 추상화된 표현도 생각해 봅시다. 

다음과 같은 방법은 어떤가요?
```suggestion
/**
 * 할 일 식별자의 생성을 담당합니다.
 */
```

Reply:
이럴땐 더 추상화된 표현이 좋겠군요. 이러면 나중에 식별하는 대상이 번호가 아닌 걸로 바뀌더라도 주석은 그대로니까요🧐

Reply:
그렇습니다!